src/main/java/apoc/index/FreeTextSearch.java

```
@@ -176,9 +196,18 @@ private void populate(Index<Node> index, Map<String, List<String>> config, Consu
         }
         try (Transaction tx = db.beginTx()) {
             updateConfigFromParameters(config, structure);
+
+            /* add options to the parameters */
+            for( String key : options.keySet() ) {
+                config.put( key, "" + options.get( key ) ); // explicit conversion to String
+            }
+            log.info("Creating or updating index '%s' with config '%s'", index, config );
             Index<Node> nodeIndex = db.index().forNodes(index, config);
             tx.success();
             return nodeIndex;
+        } catch( Exception e ) {
```

should we really hide exceptions?